<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="statusTip">
   <string/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextEdit" name="textEdit">
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="tabChangesFocus">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>621</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="newFileAction"/>
    <addaction name="openFileAction"/>
    <addaction name="saveFileAction"/>
    <addaction name="saveAsFileAction"/>
    <addaction name="printFileAction"/>
    <addaction name="separator"/>
    <addaction name="quitAction"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
    <addaction name="undoAction"/>
    <addaction name="separator"/>
    <addaction name="cutAction"/>
    <addaction name="copyAction"/>
    <addaction name="pasteAction"/>
    <addaction name="separator"/>
    <addaction name="findAction"/>
    <addaction name="findNextAction"/>
    <addaction name="replaceAction"/>
    <addaction name="changeAction"/>
    <addaction name="separator"/>
    <addaction name="allSelectAction"/>
    <addaction name="dateAction"/>
   </widget>
   <widget class="QMenu" name="menuStyle">
    <property name="title">
     <string>格式(&amp;0)</string>
    </property>
    <addaction name="autoLineAction"/>
    <addaction name="fontAction"/>
    <addaction name="colorAction"/>
   </widget>
   <widget class="QMenu" name="menuCheck">
    <property name="title">
     <string>查看(&amp;V)</string>
    </property>
    <addaction name="checkAction"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="aboutAction"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuStyle"/>
   <addaction name="menuCheck"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="newFileAction"/>
   <addaction name="openFileAction"/>
   <addaction name="saveAsFileAction"/>
   <addaction name="separator"/>
   <addaction name="cutAction"/>
   <addaction name="copyAction"/>
   <addaction name="pasteAction"/>
   <addaction name="undoAction"/>
   <addaction name="separator"/>
   <addaction name="findAction"/>
   <addaction name="allSelectAction"/>
   <addaction name="separator"/>
   <addaction name="fontAction"/>
   <addaction name="colorAction"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="newFileAction">
   <property name="icon">
    <iconset>
     <normalon>:/new/prefix1/images/Folder Open Green.ico</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>新建(&amp;N)</string>
   </property>
  </action>
  <action name="openFileAction">
   <property name="icon">
    <iconset>
     <normalon>:/new/prefix1/images/Folder Open Green.ico</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>打开(&amp;O)</string>
   </property>
  </action>
  <action name="saveFileAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/prefix1/open</normaloff>
     <normalon>:/new/prefix1/images/Folder Open Green.ico</normalon>:/new/prefix1/open</iconset>
   </property>
   <property name="text">
    <string>保存(&amp;S)</string>
   </property>
  </action>
  <action name="saveAsFileAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/prefix1/open</normaloff>
     <normalon>:/new/prefix1/images/Folder Open Green.ico</normalon>:/new/prefix1/open</iconset>
   </property>
   <property name="text">
    <string>另存为(&amp;A)</string>
   </property>
  </action>
  <action name="printFileAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/prefix1/images/Picture - BMP.ico</normaloff>:/new/prefix1/images/Picture - BMP.ico</iconset>
   </property>
   <property name="text">
    <string>打印(&amp;P)</string>
   </property>
  </action>
  <action name="quitAction">
   <property name="icon">
    <iconset>
     <normalon>:/new/prefix1/images/ffpic13061218489746_16.ico</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>退出(&amp;Q)</string>
   </property>
  </action>
  <action name="undoAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/prefix1/edit</normaloff>:/new/prefix1/edit</iconset>
   </property>
   <property name="text">
    <string>撤销(&amp;U)</string>
   </property>
  </action>
  <action name="cutAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/prefix1/edit</normaloff>:/new/prefix1/edit</iconset>
   </property>
   <property name="text">
    <string>剪切(&amp;T)</string>
   </property>
  </action>
  <action name="copyAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/prefix1/edit</normaloff>:/new/prefix1/edit</iconset>
   </property>
   <property name="text">
    <string>复制(&amp;C)</string>
   </property>
  </action>
  <action name="pasteAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/prefix1/edit</normaloff>:/new/prefix1/edit</iconset>
   </property>
   <property name="text">
    <string>粘贴(&amp;P)</string>
   </property>
  </action>
  <action name="deleteAction">
   <property name="text">
    <string>删除(&amp;L)</string>
   </property>
  </action>
  <action name="findAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/prefix1/find</normaloff>:/new/prefix1/find</iconset>
   </property>
   <property name="text">
    <string>查找(&amp;F)</string>
   </property>
  </action>
  <action name="findNextAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/prefix1/find</normaloff>:/new/prefix1/find</iconset>
   </property>
   <property name="text">
    <string>查找下一个(&amp;N)</string>
   </property>
  </action>
  <action name="replaceAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/prefix1/find</normaloff>:/new/prefix1/find</iconset>
   </property>
   <property name="text">
    <string>替换(&amp;R)</string>
   </property>
  </action>
  <action name="changeAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/prefix1/find</normaloff>:/new/prefix1/find</iconset>
   </property>
   <property name="text">
    <string>转换到(&amp;G)</string>
   </property>
  </action>
  <action name="allSelectAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/prefix1/images/ffpic1306121836412_16.ico</normaloff>:/new/prefix1/images/ffpic1306121836412_16.ico</iconset>
   </property>
   <property name="text">
    <string>全选(&amp;A)</string>
   </property>
  </action>
  <action name="dateAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/prefix1/time</normaloff>:/new/prefix1/time</iconset>
   </property>
   <property name="text">
    <string>时间/日期(&amp;D)</string>
   </property>
  </action>
  <action name="autoLineAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/prefix1/images/ffpic1306121836412_16.ico</normaloff>:/new/prefix1/images/ffpic1306121836412_16.ico</iconset>
   </property>
   <property name="text">
    <string>自动换行(&amp;W)</string>
   </property>
  </action>
  <action name="fontAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/prefix1/font</normaloff>:/new/prefix1/font</iconset>
   </property>
   <property name="text">
    <string>字体(&amp;F)</string>
   </property>
  </action>
  <action name="checkAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/prefix1/brower</normaloff>:/new/prefix1/brower</iconset>
   </property>
   <property name="text">
    <string>状态栏(&amp;S)</string>
   </property>
  </action>
  <action name="aboutAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/prefix1/brower</normaloff>:/new/prefix1/brower</iconset>
   </property>
   <property name="text">
    <string>关于记事本(&amp;A)</string>
   </property>
  </action>
  <action name="colorAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/prefix1/color</normaloff>:/new/prefix1/color</iconset>
   </property>
   <property name="text">
    <string>颜色(&amp;C)</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
